helm/event-exporter-app/templates/configmap.yaml

```diff
@@ -7,7 +7,6 @@ data:
   config.yaml: |
     logLevel: debug
 
-    {{ if .Values.Installation.V1.Secret.EventExporterApp }}
```

Comment:
What was the reason removing removing the secret here, can we assume we have it in all installation?

Reply:
With this we would add it to the default secrets: https://github.com/giantswarm/installations/pull/1519
I think that would mean it's in all installations.

Reply:
Alright that make sense, sorry missed that 🚀. Really awesome 👍